test/integration/highlight_editor_spec.mjs

```diff
@@ -1841,14 +1848,18 @@ describe("Highlight Editor", () => {
             "ternative compilation technique for dynamically-typed languages"
           );
           const editorSelector = getEditorSelector(0);
-          const x = rect.x + rect.width / 2;
+          const x = Math.round(rect.x + rect.width / 2);
           let y = rect.y + rect.height / 2;
           await page.mouse.click(x, y, { count: 3, delay: 100 });
           await page.waitForSelector(editorSelector);
           await waitForSerialized(page, 1);
           await page.keyboard.press("Escape");
           await page.waitForSelector(`${editorSelector}:not(.selectedEditor)`);
 
+          const clickHandle = await waitForPointerUp(page);
+          y = rect.y - 3 * rect.height;
+          await page.mouse.move(x, Math.round(y));
```

Comment:
To increase similarity with the other test updated above in this PR I suggest we move the `Math.round` call for `y` here to line 1852 where the actual division happens so that this block is equal.